samples/Net9FunctionApp/HelloHttp.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace Net9FunctionApp
```

Comment:
nit - should we name it as `Net9Worker` to keep it consistent with other tfm [sample apps](https://github.com/Azure/azure-functions-dotnet-worker/tree/main/samples)?

Reply:
TBH, I felt this name is more readable. In my opinion, we should update the other ones too.